tests/h/groups/__init___test.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from h.groups import includeme
+from h.groups.search import GroupAuthFilter
+
+
+def test_includeme_registers_search_filter(pyramid_config):
+    includeme(pyramid_config)
+    assert GroupAuthFilter in pyramid_config.memex_get_search_filters()
+
+
+@pytest.fixture
+def pyramid_config(pyramid_config):
+    pyramid_config.add_directive('memex_set_groupfinder', lambda c, f: None)
+
+    filters = []
+    pyramid_config.add_directive('memex_add_search_filter',
```

Comment:
Oh, right. I think I'd probably `pyramid_config.include('memex')` in preference to this, but no need to change that at the moment.